app/assets/stylesheets/goldenlayout_overwrites.less

```diff
@@ -64,9 +64,9 @@
       color: rgba(0, 0, 0, 0.65);
       font-size: 14px;
       font-weight: 400;
-      font-family: "Monospaced Number", "Chinese Quote", -apple-system, BlinkMacSystemFont,
-        "Segoe UI", Roboto, "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue",
-        Helvetica, Arial, sans-serif;
+      font-family: "Monospaced Number", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Comment:
This font caused the weird rendering of the left/right quotes. Unfortunately antd re-sets this font for some elements. Maybe we need to manually overwrite it for those cases.